README.md

```diff
@@ -26,6 +26,7 @@ Keybinding                              | Description
 <kbd>Ctrl+K</kbd>, <kbd>Shift+Tab</kbd> | Previous view.
 <kbd>Ctlr+J</kbd>, <kbd>Tab</kbd>       | Next view.
 <kbd>Ctrl+H</kbd>, <kbd>Alt+H</kbd>     | Toggle history.
+<kbd>Ctrl+E</kbd>                       | Export as cURL command
```

Comment:
We really need that custom keybinding config ability before this project become the main battlefield of emacs and vim users.. =))
I use both vim and emacs and  <kbd>ctrl+u</kbd> seems a good default to me.